Cargo.toml

```diff
@@ -22,12 +23,29 @@ structopt = "0.3.14"
 thiserror = "1.0"
 tokio = { version = "0.2.21", features = ["macros"] }
 warp = { version = "0.2", features = ["tls"] }
+diesel_migrations = "1.4.0"
+#test dependencies
+
+#diesel doesn't have an way to create empty db
```

Comment:
You mean to create a new db file?

Reply:
yes, i couldn't find any way using diesel, to create new db. while rustqlite `Connection::open` method can create new db if the file it pointing does not exist